client/control/components/ColumnMenu.js

```diff
@@ -29,6 +29,27 @@ class ColumnMenu extends React.Component {
     </span>
   );
 
+  /**
+   * Given an object, adds necessary properties to
+   * be displayed in a CheckboxList.
+   *
+   * Basically just adds a `value` property from
+   * the col's `slug`. We need this as a function due to some
+   * weird linter issue WRT anon functions returning objects.
```

Comment:
I actually figured it out shortly after while working on filters - I was trying to return an object from an anon function, like so:
`whatever={ ()=>{ my: 'object' } }`

but the linter would complain unless the returned object is wrapped in parens:
`whatever={ ()=>({ my: 'object' }) }`